### ISSUE TITLE
Torch And xformers

### ISSUE THREAD

**Issue by adabrh** [VICO-UoE/Looking3D#6]
i am trying to run the code. but facing issue with torch and xformers compatibility with python 3.8. can you tell your vesions you have used?

also python3d version.
am using Geforce rtx 4090.

**Comment by ankanbhunia** [comment]
Sorry for late reply. 

As xformers's official pip package only supports the latest torch version unless you want to install it from source. So you have to modify the version depending on what torch you are using. Our code will work any pytorch version => 1.18. 

On the other hand, pytorch3d is tricky to install. You can try: 

```
pip install "git+https://github.com/facebookresearch/pytorch3d.git"
 ```

If this does not work then you have to look for prebuilt packages (only available for specific python/cuda/torch version) that are compatiable.  Maybe you can create a seperate conda environment to match the reuqired python/cuda/torchversion and. 

Note pytorch3d is not required to perform training and testing. It is only reauired to render multi-view images.

**Comment by ankanbhunia** [comment]
As the issue has been active for a long time, I am closing it for now. Please reopen it if required.

**Comment by adabrh** [comment]
Hey.
If i want to know which chair image belongs to which label in ShapeNet.

For Example:
ShapeNet\ {c (class)} \ {label}

Here Our Class is chair. I want to know how to align labels of shapenet with brokenchair dataset.
Kindly help in this matter.
